features/user-associated-with-multiple-accounts.md

```diff
@@ -0,0 +1,26 @@
+# User associated with multiple accounts
+
+A user can be associated with multiple ImageKit accounts using the same email address. A previously registered user can be added to a different account. Read how to add a user to an account [here](./user-access-management/README.md).
+
+The login flow for a multi-account user is as follows:
+
+At the login screen, you will be asked to enter your email address and password (any valid password for one of the accounts).
+![Login screen](../.gitbook/assets/login_screen.png)
+
+Then you'll be sent to the next page and prompted to enter a one-time password (OTP) that was sent to the associated email address.
```

Comment:
On clicking "Submit", a one-time password (OTP) will be sent to your registered email address. You will be asked to enter this OTP on the next screen.

Reply:
done